OutOfSchool/OutOfSchool.WebApi.Tests/Services/BlockedProviderParentServiceTests.cs

```diff
@@ -171,25 +173,43 @@ public async Task Block_WhenDtoIsAlreadyBlocked_ShouldReturnFailed()
     public async Task Unblock_WhenDtoAndUserIdIsValid_ShouldReturnSuccess()
     {
         // Arrange
-        var parent = ParentGenerator.Generate();
-        parent.Id = unblockDto.ParentId;
+        var parent = ParentGenerator.Generate().WithUserId(userId);
+        parent.Id = blockDto.ParentId;
+        unblockEntity.Parent = parent;
+        var unblockedParentUserId = Guid.Parse(parent.UserId);
+        var additionalData = new Dictionary<string, string>()
+        {
+            { ProviderIdKey, unblockEntity.ProviderId.ToString() },
+            { ProviderFullTitleKey, unblockEntity.Provider.FullTitle },
+            { ProviderShortTitleKey, unblockEntity.Provider.ShortTitle },
+        };
+
         var blockedParents = new List<BlockedProviderParent>
         {
             unblockEntity,
         };
 
         blockedProviderParentRepositoryMock
-            .Setup(x => x.GetByFilter(It.IsAny<Expression<Func<BlockedProviderParent, bool>>>(), It.IsAny<string>()))
-            .ReturnsAsync(blockedParents);
+            .Setup(x => x.GetBlockedProviderParentEntities(It.IsAny<Guid>(), It.IsAny<Guid>()))
```

Comment:
(Test readability) Please whenever possible replace `It.IsAny` with expected parameters - here they are (I guess) `unblockEntity.ParentId` and `unblockEntity.ProviderId`. Similar in other tests

Reply:
I've replaced them. Thank you!